src/components/Categories/GetCategories.vue

```diff
@@ -1,55 +1,72 @@
 <template>     
-    <div class = "container">  
-		<h2 :style="{textAlign:'center', marginTop:'30px', marginBottom:'30px'}">Danh sách tác giả</h2>
-		<table class="table">  
-			<thead>  
-				<th :style="{paddingLeft:'8px'}">Id</th>  
-				<th :style="{paddingLeft:'8px'}">Thể loại</th>  
-				<th :style="{paddingLeft:'8px'}">Mô tả</th>  
-				<th></th>  
-			</thead>  
-			<tbody>  
-				<tr v-for="item in categories" :key="item.id">  
-					<td>{{ item.id }}</td>  
-					<td>{{ item.name }}</td>  
-					<td>{{ item.description }}</td>  
-					<td><input class="btn btn-success" type='button' value='Chi tiết' @click="detail(item.id)"/></td>
-				</tr>  
-			</tbody>  
-		</table>  
+    <div >  
+		<h2 :style="{textAlign:'center', marginTop:'30px', marginBottom:'30px'}">Danh sách thể loại</h2>
+			<DataTable :value="categories">
+				<Column field="id" header="ID" style="width: 70px"/>
+
+				<Column field="name" header="Thể loại" style="width: 200px"/>
+
+				<Column field="description" header="Mô tả" style="width: 550px"/>
+
+				<Column field="id" header="Chi tiết">
+					<template #body="slotProps" >
+						<Button type="button" @click="detail(slotProps.data.id)" label="Xem" class="btn btn-detail"></Button>
+					</template>
+				</Column>
+			</DataTable>
     </div>  
 </template>  
 
 <script>
 import categoryServices from '@/services/categories.services';
+import DataTable from 'primevue/datatable';
+import Column from 'primevue/column';
+import router from 'vue-router'
 export default {  
 	name: 'GetCategories',  
-		data() {
-			return {
-				categories: [],
-			};
-		},
-		created()  
-		{  
-			this.getCategories();  
-		},  
+	data() {
+		return {
+			categories: [],
+		};
+	},
+
+	created()  
+	{  
+		this.getCategories(); 
+	},  
+
+	components: {
+		DataTable,
+		Column
+	},
 	
 	methods: { 
 		getCategories() {
             categoryServices.getCategories().then(response =>
-                this.categories = response.data,
+                {
+					this.categories = response.data
+				}
             ).catch(e => {  
 					console.log(e);  })
 		},
 		detail(categoryId){
-			this.$router.push('/getcategorybyid/' + categoryId); 
-		}
+			this.$router.push('/getcategory/' + categoryId); 
+		},
 
 	}  
 } 
 </script>
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
-
+	.btn-detail {
+	color: #fff;
+	background-color: #198754;
+	border-color: #198754;
+	}
+	.btn-detail:hover {
+		color: #fff;
+		background-color: #157347;
+		border-color: #146c43;
+	}
```

Comment:
Mấy cái này hoàn toàn không cần thiết, thư viện làm hết rồi

Reply:
có vài cái nó bị đè bởi 1 số file css khác nên em define lại luôn để mấy dòng này nó hoạt động được